<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#660099;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>581</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../image/Logo-design-Width.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>90</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>RUBIK 4X4X4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btngan460">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>181</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../image/vn-11134207-7qukw-lh02j8n9wt8345_resize_w450_nl-removebg-preview.png</normaloff>../image/vn-11134207-7qukw-lh02j8n9wt8345_resize_w450_nl-removebg-preview.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>180</width>
      <height>180</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btngandep">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>250</y>
      <width>161</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../image/sg-11134201-7ravc-ma58cdhhhrcv44-removebg-preview.png</normaloff>../image/sg-11134201-7ravc-ma58cdhhhrcv44-removebg-preview.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>180</width>
      <height>180</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnganlo">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>240</y>
      <width>181</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../image/b0f951fa642f3a271983aa080c82c6c8-removebg-preview.png</normaloff>../image/b0f951fa642f3a271983aa080c82c6c8-removebg-preview.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>180</width>
      <height>180</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btncb">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>500</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
    <property name="text">
     <string>Quay láº¡i</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>802</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
